src/components/BookList.js

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable func-names */
+/* eslint-disable react/forbid-prop-types */
+/* eslint-disable react/jsx-key */
+/* eslint-disable react/jsx-filename-extension */
+/* eslint-disable react/destructuring-assignment */
+/* eslint-disable react/prefer-stateless-function */
+import React from 'react';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import Actions from '../actions';
+import Book from './Book';
+
+
+const mapStateToProps = function (state) {
+  return {
+    bookStore: state,
+  };
+};
+
+const mapDispatchToProps = function (dispatch) {
+  return bindActionCreators({
+    createBook: Actions.createBook,
+    removeBook: Actions.removeBook,
+  }, dispatch);
+};
+
+class BookList extends React.Component {
+  render() {
+    const booksAll = this.props.bookStore.bookestore.map(item => <Book book={item} />);
```

Comment:
hello, instead of doing `this.props.bookStore.bookestore.map` lets get that in the mapStateToProps 
```
const mapStateToProps = function (state) {
  return {
    bookStore: statebookstore,
  };
};
```

[OPTIONAL]